debian/changelog

```diff
@@ -1,3 +1,10 @@
+azmi (0.4.1-1) stable; urgency=medium
+
+  * Bug fix - azmi enters endless loop if no identity on the VM
+  * Full list of issues: https://github.com/SRE-PRG/azmitool/milestone/6?closed=1
+
+ -- SRE-PRG <NA@NA.com>  Fri, 19 Jun 2020 13:13:04 +0200
```

Comment:
To consider remove SRE-PRG and any GitHub group email that exists for authors of this project?